src/main/jbake/content/creating-example002.adoc

```diff
@@ -45,7 +45,7 @@ repository by following these steps:
 
 1.  In NetBeans IDE select *File*, then *Open Project*, navigate to the
 location where you installed the tutorial (usually
-`glassfish5/docs/firstcup`), select `example`, deselect the *Open
+`glassfish6/docs/firstcup`), select `example`, deselect the *Open
```

Comment:
Currently the tutorial and first-cup are not bundled with glassfish. So this instruction needs to be changed.